src/main/java/br/com/excaladashboard/models/Funcionarios.java

```diff
@@ -0,0 +1,132 @@
+package br.com.excaladashboard.models;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "funcionarios")
+public class Funcionarios {
```

Comment:
Tirar o plural em classes

```suggestion
public class Funcionario {
```